crates/bevy_core/src/time/time.rs

```diff
@@ -1,6 +1,18 @@
-use bevy_ecs::system::ResMut;
+use async_channel::{Receiver, Sender};
+use bevy_ecs::system::{Res, ResMut};
 use bevy_utils::{Duration, Instant};
 
+/// channel resource used to receive time from render world
+pub struct TimeReceiver(pub Receiver<Instant>);
+/// channel resource used to send time from render world
+pub struct TimeSender(pub Sender<Instant>);
```

Comment:
Is there a reason why the internal value is pub? Probably worth not making these pub and exposing a `send` on the sender.

Reply:
I considered these just new typed channels. I didn't really want to limit what functions users might want to call on these in case they're using the channels for their own time update logic.